README.md

```diff
@@ -30,7 +30,7 @@ See the [adoption guides](https://jupyter-releaser.readthedocs.io/en/latest/how_
 
 GitHub actions scripts are available to draft a changelog, draft a release, publish a release, and check a release.
 
-See the [action details documentation](https://jupyter-releaser.readthedocs.io/en/latest/background/theory.html#action-details) for more information.
+See the [action details documentation](https://jupyter-releaser.readthedocs.io/en/latest/reference/theory.html#action-details) for more information.
```

Comment:
The `check-links` failure is expected, and should resolve itself after merging:

![image](https://github.com/user-attachments/assets/56e185f4-3739-4607-abc1-292d562e9ec7)